### DISCUSSION TITLE
[Question] Layers outer most layer is called `section` instead of `body`

### DISCUSSION THREAD

**Discussion by iamkeiryan** [GrapesJS/grapesjs#6298] [Q&A] [answered]
Hi, 

I checked the documentation and using the default layers manager but when it loads, the outer most layer is called `section` instead of body.

Here's the configuration that I've used:
`layerManager: {
          appendTo: '#layers-cont',
          sortable: false,
          hidable: true,
          showHover: false,
          showWrapper: false,
          custom: true,
 },`

![image](https://github.com/user-attachments/assets/009a7e96-5c4d-477d-a5be-79c749ff1d4b)

Any ideas what do I need to change to show the body as the outer most layer?

**Comment by artf** [comment, marked as answer]
remove `showWrapper: false`
